app/screens/channel/channel_post_list/channel_post_list.js

```diff
@@ -77,14 +81,14 @@ export default class ChannelPostList extends PureComponent {
     }
 
     getVisiblePostIds = (props) => {
-        return props.postIds.slice(0, props.postVisibility);
+        return props.postIds ? props.postIds.slice(0, props.postVisibility) : [];
```

Comment:
you are setting the default prop to `[]` so postIds is defined here, is your intent to check for the length?

Reply:
That's what I had been trying to do, but the problem is that default props are only applied when the prop not defined. I found that the selector returns `null` which doesn't trigger the default props to be used.

Reply:
ah, the returned `null` in the selector again :S